README.md

```diff
@@ -11,75 +11,184 @@ Generative AI is fallible in ways similar to people. It makes typos, misintepret
 falls down rabbit holes. We already have tools to help fallible individuals code together: issues, pull requests,
 and code reviews. Let's apply those same tools to collaborate with a new breed of intelligence.
 
-## Setup
+## Installation
 
-### GitHub User Setup
+### Option 1: GitHub Action (Recommended)
 
-1. Create a new GitHub user account for your bot
+The easiest way to run the bot is as a GitHub Action that automatically responds to assigned issues and PR comments.
+
+#### Prerequisites
+
+1. **Create a Bot GitHub Account**:
```

Comment:
Creating a GitHub account for the bot is required regardless of deployment mechanism, so let's move this to a section separate from the different deployment options.

Same for generating the github and anthropic tokens